taxcalc/tests/test_policy.py

```diff
@@ -248,51 +243,6 @@ def test_multi_year_reform():
     wfactor = {}
     for i in range(0, nyrs):
         wfactor[syr + i] = 1.0 + wratelist[i]
-    # confirm that parameters have current-law values
-    assert np.allclose(getattr(pol, '_EITC_c'),
-                       Policy._expand_array(
-                           np.array([[487, 3250, 5372, 6044],
-                                     [496, 3305, 5460, 6143],
-                                     [503, 3359, 5548, 6242],
-                                     [506, 3373, 5572, 6269],
-                                     [510, 3400, 5616, 6318],
-                                     [519, 3461, 5716, 6431],
-                                     [529, 3526, 5828, 6557]],
-                                    dtype=np.float64),
-                           'real',
-                           inflate=True,
-                           inflation_rates=iratelist,
-                           num_years=nyrs),
-                       atol=0.01, rtol=0.0)
-    assert np.allclose(getattr(pol, '_STD_Dep'),
-                       Policy._expand_array(
-                           np.array([1000, 1000, 1050, 1050, 1050, 1050, 1100],
-                                    dtype=np.float64),
-                           'real',
-                           inflate=True,
-                           inflation_rates=iratelist,
-                           num_years=nyrs),
-                       atol=0.01, rtol=0.0)
-    assert np.allclose(getattr(pol, '_CTC_c'),
-                       Policy._expand_array(
-                           np.array([1000] * 5 + [2000] * 8 + [1000],
-                                    dtype=np.float64),
-                           'real',
-                           inflate=False,
-                           inflation_rates=iratelist,
-                           num_years=nyrs),
-                       atol=0.01, rtol=0.0)
-    # this parameter uses a different indexing rate
-    assert np.allclose(getattr(pol, '_SS_Earnings_c'),
-                       Policy._expand_array(
-                           np.array([113700, 117000, 118500, 118500, 127200,
-                                     128400, 132900],
-                                    dtype=np.float64),
-                           'real',
-                           inflate=True,
-                           inflation_rates=wratelist,
-                           num_years=nyrs),
-                       atol=0.01, rtol=0.0)
```

Comment:
`_expand_array` is no longer used, and loading values correctly is already tested extensively by ParamTools.